Sources/StreamChat/Database/DTOs/BlockedUserDTO.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+import Foundation
+
+@objc(BlockedUserDTO)
+class BlockedUserDTO: NSManagedObject {
+    @NSManaged var blockedUserId: String
+    @NSManaged var blockedAt: DBDate?
+    
+    func asModel() throws -> BlockedUser {
+        .init(
+            userId: blockedUserId,
+            blockedAt: blockedAt?.bridgeDate ?? .init()
```

Comment:
The blockedAt is nullable right?
```suggestion
            blockedAt: blockedAt?.bridgeDate
```